documents/Libraries/pbrlib/genglsl/mx_multiply_bsdfC.glsl

```diff
@@ -0,0 +1,14 @@
+void mx_multiply_bsdfC_reflection(vec3 L, vec3 V, BSDF in1, vec3 in2, out BSDF result)
```

Comment:
What would you think of the alternate naming convention "mx_multiply_bsdf_color" and "mx_multiply_bsdf_float"?  It's a bit more verbose, but potentially clearer to a new reader.

Reply:
Sounds good, I'll make the change.